<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DollarsDollarsDollarsClass</class>
 <widget class="QMainWindow" name="DollarsDollarsDollarsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DollarsDollarsDollars</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="billListWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="sortPushButton">
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="calculateTotalPushButton">
          <property name="text">
           <string>Calculate total</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="companyNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="unpaidBillsLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DollarsDollarsDollars.qrc"/>
 </resources>
 <connections/>
</ui>
